crates/cxx-qt-gen/src/generator/rust/cxxqttype.rs

```diff
@@ -8,57 +8,62 @@ use crate::{
     naming::TypeNames,
 };
 use quote::quote;
-use syn::Result;
+use syn::{Ident, Result};
 
 use super::fragment::RustFragmentPair;
 
 pub fn generate(
     qobject_ident: &QObjectNames,
     type_names: &TypeNames,
+    module_ident: &Ident,
 ) -> Result<GeneratedRustFragment> {
     let mut blocks = GeneratedRustFragment::default();
 
     let cpp_struct_ident = &qobject_ident.name.rust_unqualified();
     let rust_struct_ident = &qobject_ident.rust_struct.rust_unqualified();
+    let rust_fn = qobject_ident.cxx_qt_ffi_method("rust");
+    let rust_mut_fn = qobject_ident.cxx_qt_ffi_method("rust_mut");
```

Comment:
should we have these as const's somewhere rather than magic values?

Reply:
Hm, we don't do that anywhere else, so don't know why we should start here :thinking: 
After the refactor, these are the names of the functions in C++, so the compiler will check that they match.

Reply:
Right can be done separately, just wondered if we have these magic values used in multiple parts of the generation if they should be a const static somewhere